src/hooks/use-ordinals.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable no-await-in-loop */
+import { useMemo, useState } from "react";
+import { FETCH_SIZE } from "@lib/constants";
+import { useEffectOnce, useDebounce } from "react-use";
+import { matchSorter } from "match-sorter";
+
+const axios = require("axios");
```

Comment:
Why are we importing this the old way, 
```suggestion
import axios from 'axios'
```